frontend/src/pages/projects/notebook/StartNotebookModal.tsx

```diff
@@ -164,7 +164,7 @@ const StartNotebookModal: React.FC<StartNotebookModalProps> = ({
             <List isPlain isBordered data-id="event-logs">
               {events
                 .slice()
-                .reverse()
+                .toSorted((a, b) => getEventTimestamp(b).localeCompare(getEventTimestamp(a)))
```

Comment:
timestamps really should be sorted like this

```tsx
new Date(a) - new Date(b)
```

otherwise you are comparing strings which may not always be accurate